src/components/v5/common/ActionSidebar/partials/forms/ManageColonyObjectivesForm/consts.ts

```diff
@@ -13,9 +13,7 @@ export const validationSchema = object()
       .trim()
       .max(MAX_ANNOTATION_LENGTH)
       .required(() => formatText({ id: 'errors.colonyObjective.description' })),
-    colonyObjectiveProgress: number()
-      .max(100)
-      .required(() => formatText({ id: 'errors.colonyObjective.progress' })),
+    colonyObjectiveProgress: number().max(100).notRequired(),
```

Comment:
dropping a comment just in case if this is a straggler :D

Reply:
Jesus man, how did you catch this? You are scaring me